src/language-js/parse/postprocess/index.js

```diff
@@ -154,6 +157,27 @@ function postprocess(ast, options) {
     }
   });
 
+  /* eslint-disable prettier-internal-rules/no-node-comments */
+  if (isNonEmptyArray(ast.comments)) {
+    let followingComment = getLast(ast.comments);
+    for (let i = ast.comments.length - 2; i >= 0; i--) {
+      const comment = ast.comments[i];
+      if (
+        locEnd(comment) === locStart(followingComment) &&
+        isBlockComment(comment) &&
+        isIndentableBlockComment(comment) &&
+        isBlockComment(followingComment) &&
+        isIndentableBlockComment(followingComment)
```

Comment:
```suggestion
        isBlockComment(comment) &&
        isBlockComment(followingComment) &&
        isIndentableBlockComment(comment) &&
        isIndentableBlockComment(followingComment)
```

Let's move `isBlockComment` check up, to avoid split big comments.